### PR TITLE
#135842519 add karma and jasmine to setup frontend testing

### PR DESCRIPTION
**Pull request by jkanyua**
#### What does this PR do?
 It adds Karma to setup front end testing environment.

 #### Description of Task to be completed?
It adds Jasmine and Karma(with PhantomJS, Chrome and Firefox) to setup front end testing environment.

 #### How should this be manually tested?
run:
```
karma start
```
 #### Any background context you want to provide?
 #### What are the relevant pivotal tracker stories?
#135842519
 #### Screenshots (if appropriate)
 #### Questions:

### CODE REVIEW

**Review by boswellgathu — CHANGES_REQUESTED** [review]
Check on the wiki on how to create a pull request, including the heading

**Comment by gngchrs** [comment]
@andela-jkanyua You have conflicts.

**Comment by gngchrs** [comment]
@andela-jkanyua I see the smoke-test, but it has not yet been included in our continuous deployment. I think you still have to manually run Karma to run the tests. Work with @andela-cwekesa to see how to add it to the gulp tasks, or one of the npm scripts. It should also include coverage, so @andela-Bgathu has to aslo chip in.

**Review by GathuBoswell — APPROVED** [review]

File: karma.conf.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
+
+
+    // list of files / patterns to load in the browser
+    files: [
+    './test/test-spec.js',
```
Comment (jkanyua) [inline karma.conf.js:18]: test should be last

**Comment by gngchrs** [comment]
@andela-jkanyua please make the changes we talked about when you can and reopen this with a reviewer.

**Pull request closed without merging**